fix some syntax error run in python3

MYLF opened the pull request:
print is a function in python3 , we should using print() to make it available in python3

atacms commented on .gitignore:
```diff
@@ -1,233 +1,15 @@
-#############
-## testing bigworld model files
```
let's keep this header

atacms commented on .gitignore:
```diff
@@ -1,233 +1,15 @@
 *.fbx
 *.dae
 *.max
-*.txt
```
I do need txt and rar exclusions. I used them to pack resources to other wot modders.

atacms commented:
r're quite diligent. Many of the odd styled syntax is the legacy of SkaceKamen. But since it works I didn't bother to change them.

MYLF committed:
fix some syntax error run in python3

MYLF committed:
clean up

MYLF committed:
Update README.md

MYLF committed:
folder input function with -gui

now,you can input a folder when used -gui,converter will find all primitives file in your folder and unpack them

MYLF committed:
py2x bug fix
